code_examples/sdk_examples/src/core_features/getting_started/05_fetch_endpoint_data.ts

```diff
@@ -1,13 +1,12 @@
 import axios from 'axios'
 
 import * as Kilt from '@kiltprotocol/sdk-js'
+import { VerifiableCredential } from '@kiltprotocol/credentials/lib/cjs/V1/types'
```

Comment:
```suggestion
import { types } from '@kiltprotocol/credentials'
```

and then use `types.VerfiableCredential`. But we may want to move that to that /types package, doesn't seem right to have to access it that way

Reply:
Its fix on: https://github.com/KILTprotocol/docs/pull/329/commits/2fb4bf229be1bd8c28ed45066790a66d8b5f1e29
But types declaration works partially, even tho it connects the modules in types, its also says `'"@kiltprotocol/credentials"' has no exported member named 'types'. Did you mean 'Types'?ts(2724)`

Reply:
ah yes it may have been `Types`